hyungjoonYoon/app.js

```diff
@@ -49,7 +49,109 @@ app.post("/signUp", async (req, res, next) => {
     `,
     [name, email, profile_image, password]
   );
-  res.status(201).json({ message: "successfully created" });
+  res.status(201).json({ message: "userCreated" });
+});
+
+app.post("/post", async (req, res) => {
+  const { title, content, user_id } = req.body;
+  await appDataSource.query(
+    `INSERT INTO posts(
+      title,
+      content,
+      user_id
+    ) 
+    VALUES (?,?,?);
+    `,
+    [title, content, user_id]
+  );
+  res.status(201).json({ message: "postCreated" });
+});
+
+app.get("/lookUpPosts", async (req, res) => {
+  await appDataSource.query(
+    `SELECT 
+    u.id AS userId, 
+    u.profile_image AS userProfileImage,
+    p.id AS postingId,
+    p.postingImageUrl AS postingImageUrl,
+    p.content AS postingContent
+    FROM posts p
+    INNER JOIN users AS u
+    ON u.id = p.user_id;`,
+    (err, rows) => {
+      res.status(200).json({ data: rows });
+    }
+  );
+});
```

Comment:
- RESTful api는 엔드포인트를 동사형이 아닌 명사형으로 작성하도록 권장합니다. HTTP GET 메서드 자체는 리소스를 조회하는 의미를 포함하고 있기 때문에 lookup이라는 동사는 빼주세요!
- sql문의 가독성을 위해 indentation(들여쓰기)을 수정해주세요!
```suggestion
app.get("/posts", async (req, res) => {
  await appDataSource.query(
    `SELECT 
        u.id AS userId, 
        u.profile_image AS userProfileImage,
        p.id AS postingId,
        p.postingImageUrl AS postingImageUrl,
        p.content AS postingContent
      FROM posts AS p
      INNER JOIN users AS u ON u.id = p.user_id;`,
    (err, rows) => {
      res.status(200).json({ data: rows });
    }
  );
});
```